packages/workspace/src/common/workspace-protocol.ts

```diff
@@ -32,4 +32,9 @@ export interface WorkspaceServer {
      */
     setRoot(uri: string): Promise<void>;
 
+    /**
+     * Returns list of recents opened workspace as a string []
```

Comment:
recents this should be recent or recently opened or anything else w/o the typo.
Also, can we go with `array` instead or `[]` for the docs?

Reply:
OK fixed the typo